src/LiveDevelopment/Agents/RemoteFunctions.js

```diff
@@ -268,21 +269,15 @@ function RemoteFunctions(experimental) {
                 "box-shadow": "0 0 1px #fff",
                 "box-sizing": "border-box"
             };
+            
+            var mergedStyles = Object.assign({}, stylesToSet,  config.remoteHighlight.stylesToSet);
```

Comment:
You mean a real browser and not Brackets itself, right? (Not much familiar with LivePreview)
Then I suppose we can use `Object.assign`. Sorry for the noise...